html/changelogs/Eyeveri_VaultTweaks.yml

```diff
@@ -0,0 +1,43 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ReadThisNamePlz
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixes the Cameras in the vault not having a motion alarm"
+  - maptweak: "Tweaks the vault to have two walls instead of one. Adds two additional turrets. Adds a emergency O2 wall locker instead of the standing locker to ops."
+  - balance: "Readjusts some of the loot and adds less valuable loot possibilities for the safe."
```

Comment:
```suggestion
  - bugfix: "Fixed the cameras in the vault not having a motion alarm."
  - maptweak: "Tweaked the vault to have two-thick walls instead of one. Added two additional turrets. Added a emergency O2 wall locker instead of the standing locker to ops."
  - balance: "Readjusted some of the loot and adds less valuable loot possibilities for the safe."
```